src/lib/wdi5-bridge.ts

```diff
@@ -257,7 +257,7 @@ export async function addWdi5Commands() {
         const oRoute = oOptions.oRoute
 
         if (sHash && sHash.length > 0) {
-            const url = (browser.config as wdi5Config).wdi5["url"]
+            const url = (browser.config as wdi5Config).wdi5["url"]  || await browser.getUrl();
```

Comment:
please apply `prettier` formatting 
(this means essentially removing the ";") 🤣